docs/topics/api/reviews.rst

```diff
@@ -0,0 +1,46 @@
+=======
+Reviews
+=======
+
+.. note::
+
+    These APIs are experimental and are currently being worked on. Endpoints
+    may change without warning. The only authentication method available at
+    the moment is :ref:`the internal one<api-auth-internal>`.
+
+----
+List
+----
+
+.. review-list:
+
+This endpoint allows you to fetch user reviews for a given add-on.
+
+.. http:get:: /api/v3/addons/addon/(int:id|string:slug|string:guid)/reviews/(int:id)/
+
+    :>json int count: The number of results for this query.
+    :>json string next: The URL of the next page of results.
+    :>json string previous: The URL of the previous page of results.
+    :>json array results: An array of :ref:`reviews <review-detail-object>`.
+
```

Comment:
On second thought... I've looked at sphinx-httpdomain docs and most examples omit `param` because it's already in the URL (the type of the parameter is even included already). If I were to add the `param` directives it'd look a bit redundant indeed:
```
    :param int id: The add-on id.
    :param string slug: The add-on slug.
    :param string guid: The add-on guid.
```

And there is no good way to show that these 3 are linked together and you should only provide one.